packages/react-app/src/components/bridge/UnlockButton.jsx

```diff
@@ -41,7 +41,24 @@ export const UnlockButton = () => {
     if (!unlockLoading && !allowed && valid()) {
       approve().catch(error => {
         if (error && error.message) {
-          showError(error.message);
+          if (error.data === 'Bad instruction fe') {
```

Comment:
It is not correct to rely only on such message. Because it is specific to usage of the `assert` function in the Solidity code. Another token could use `revert` instead of `assert` to identify the same case but the error will be different.